docs/interpreter/postgresql.md

```diff
@@ -1,230 +1,28 @@
-
-```sql
-%psql.sql
-select count(*) from mytable;
-select * from mytable;
-```
-
-### PSQL command line tools
-Use the Shell Interpreter (`%sh`) to access the command line [PSQL](http://www.postgresql.org/docs/9.4/static/app-psql.html) interactively:
-
-```bash
-%sh
-psql -h phd3.localdomain -U gpadmin -p 5432 <<EOF
- \dn  
- \q
-EOF
-```
-
-This will produce output like this:
-
-```
-        Name        |  Owner  
---------------------+---------
- hawq_toolkit       | gpadmin
- information_schema | gpadmin
- madlib             | gpadmin
- pg_catalog         | gpadmin
- pg_toast           | gpadmin
- public             | gpadmin
- retail_demo        | gpadmin
-```
-
-### Apply Zeppelin Dynamic Forms
-You can leverage [Zeppelin Dynamic Form](../manual/dynamicform.html) inside your queries. You can use both the `text input` and `select form` parametrization features
-
-```sql
-%psql.sql
-SELECT ${group_by}, count(*) as count
-FROM retail_demo.order_lineitems_pxf
-GROUP BY ${group_by=product_id,product_id|product_name|customer_id|store_id}
-ORDER BY count ${order=DESC,DESC|ASC}
-LIMIT ${limit=10};
-```
-
-### Example HAWQ PXF/HDFS Tables
-Create HAWQ external table that read data from tab-separated-value data in HDFS.
-
-```sql
-%psql.sql
-CREATE EXTERNAL TABLE retail_demo.payment_methods_pxf (
-  payment_method_id smallint,
-  payment_method_code character varying(20)
-) LOCATION ('pxf://${NAME_NODE_HOST}:50070/retail_demo/payment_methods.tsv.gz?profile=HdfsTextSimple') FORMAT 'TEXT' (DELIMITER = E'\t');
-```
-
-And retrieve content
-
-```sql
-%psql.sql
-select * from retail_demo.payment_methods_pxf
-```
-
-## Auto-completion
-The PSQL Interpreter provides a basic auto-completion functionality. On `(Ctrl+.)` it list the most relevant suggestions in a pop-up window. In addition to the SQL keyword the interpreter provides suggestions for the Schema, Table, Column names as well.
+---		
+layout: page		
+title: "PostgreSQL, Apache HAWQ (incubating) Interpreter for Apache Zeppelin"		
+description: "Apache Zeppelin supports PostgreSQL, Apache HAWQ(incubating) and Greenplum SQL data processing engines."		
+group: interpreter		
+---		
+<!--		
+Licensed under the Apache License, Version 2.0 (the "License");		
+you may not use this file except in compliance with the License.		
+You may obtain a copy of the License at		
+		
+http://www.apache.org/licenses/LICENSE-2.0		
+		
+Unless required by applicable law or agreed to in writing, software		
+distributed under the License is distributed on an "AS IS" BASIS,		
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.		
+See the License for the specific language governing permissions and		
+limitations under the License.		
+-->		
+{% include JB/setup %}		
+		
+# PostgreSQL, Apache HAWQ (incubating) Interpreter for Apache Zeppelin		
+		
+<div id="toc"></div>		
+		
+## Important Notice		
+		
+Postgresql Interpreter is deprecated and merged into [JDBC Interpreter](./jdbc.md).
```

Comment:
The link should be `./jdbc.html`. And how about pointing Postgresql 's setting example in JDBC interpreter page? e.g. 

```Postgresql interpreter is deprecated and merged into [JDBC Interpreter](./jdbc.html). You can use it with JDBC Interpreter as same functionality. See [Postgresql setting example](./jdbc.html#postgres) for more detailed information.```